calico_containers/calicoctl.py

```diff
@@ -25,7 +25,7 @@
   calicoctl container <CONTAINER> ip (add|remove) <IP> [--interface=<INTERFACE>]
   calicoctl container add <CONTAINER> <IP> [--interface=<INTERFACE>]
   calicoctl container remove <CONTAINER> [--force]
-  calicoctl endpoint show [--endpoint=<ENDPOINT_ID>] [--workload=<WORKLOAD_ID>] [--orchestrator=<ORCHESTRATOR_ID>] [--host=<HOSTNAME>] [--detailed]
+  calicoctl endpoint show [--host=<HOSTNAME>] [--orchestrator=<ORCHESTRATOR_ID>] [--workload=<WORKLOAD_ID>] [--endpoint=<ENDPOINT_ID>] [--detailed]
```

Comment:
I reordered these to make the ordering consistant:

hostname, orchestrator_id, workload_id, endpoint_id